web/src/page_params.ts

```diff
@@ -35,17 +42,21 @@ export const page_params: {
     request_language: string;
     server_avatar_changes_disabled: boolean;
     server_name_changes_disabled: boolean;
+    server_needs_upgrade: boolean;
     server_sentry_dsn: string | undefined;
     server_sentry_environment: string | undefined;
     server_sentry_sample_rate: number | undefined;
     server_sentry_trace_rate: number | undefined;
     server_web_public_streams_enabled: boolean;
+    show_billing: boolean;
+    show_plans: boolean;
+    show_webathena: boolean;
     translation_data: Record<string, string>;
     user_id: number | undefined;
     webpack_public_path: string;
     zulip_plan_is_not_limited: boolean;
+    zulip_merge_base: string;
     zulip_version: string;
-    muted_users: {id: number; timestamp: number}[];
```

Comment:
Looks like there are some reorderings going on at the same time with additions of new params. This makes it harder to track what properties are added in the diff. It would be better if this can be done separately by splitting the reorderings into a separate commit.

Reply:
Okay, fixing it.

Reply:
@PIG208 done